### PR TITLE
remove hyperdiffusion of grid-scale w

### PR DESCRIPTION
**Pull request by szy21**
## Purpose 

Remove hyperdiffusion of grid-scale vertical velocity as we are not sure if it is correct, especially when there is topography. Fow now we keep only the hyperdiffusion of subgrid-scale vertical velocity.

This partially reverts #1605 (reverts the change to grid-scale w but keeps the change to subgrid-scale w).

## Content

- Remove hyperdiffusion of grid-scale vertical velocity @akshaysridhar 
- Update docs
- Change back the dt in some SSP longrun jobs @valeriabarra  

----
- [ ] I have read and checked the items on the review checklist.

### CODE REVIEW

**Comment by valeriabarra** [comment]
Thanks! Is this PR exactly the reverse of #1605 ? If so, it might be good to mention that in the PR description, for future reference.

**Comment by szy21** [comment]
> Thanks! Is this PR exactly the reverse of #1605 ? If so, it might be good to mention that in the PR description, for future reference.

Not exactly, it reverses part of #1605. I will add it to the description.

**Comment by jiahe23** [comment]
;)

**Review by jiahe23 — APPROVED** [review]

File: regression_tests/ref_counter.jl [hunk 1]
```diff
@@ -1 +1 @@
-99
+100
```
Comment (trontrytel) [inline regression_tests/ref_counter.jl:1]: 🥇

Comment (glwagner) [reply regression_tests/ref_counter.jl:1]: 💯

Comment (szy21) [reply regression_tests/ref_counter.jl:1]: 😀

**Pull request merged**